<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auger Electron Analyzer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>KE</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="KE_doubleSpinBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Work Func</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="work_func_doubleSpinBox"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="multiplier_checkBox">
        <property name="text">
         <string>Multiplier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Retarding Mode</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="retarding_mode_comboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>CRR Ratio</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Pass Energy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="crr_ratio_doubleSpinBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="pass_energy_doubleSpinBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="resolution_doubleSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Quadrupoles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="qX1_doubleSpinBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="qY1_doubleSpinBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>X2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Y2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QDoubleSpinBox" name="qX2_doubleSpinBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="qY2_doubleSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <tabstops>
  <tabstop>multiplier_checkBox</tabstop>
  <tabstop>KE_doubleSpinBox</tabstop>
  <tabstop>work_func_doubleSpinBox</tabstop>
  <tabstop>retarding_mode_comboBox</tabstop>
  <tabstop>crr_ratio_doubleSpinBox</tabstop>
  <tabstop>pass_energy_doubleSpinBox</tabstop>
  <tabstop>resolution_doubleSpinBox</tabstop>
  <tabstop>qX1_doubleSpinBox</tabstop>
  <tabstop>qY1_doubleSpinBox</tabstop>
  <tabstop>qX2_doubleSpinBox</tabstop>
  <tabstop>qY2_doubleSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
